runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,56 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version; class=%2$.*1$s, offset=%3$u, major.minor=%4$u.%5$u, maximum major value=%6$u.0
```

Comment:
I'd like to see better consistency of these new messages:
* I suggest they should all begin with "... version % of class %"
* the offset should appear at the end of each message "; offset=%"